src/Classes/Particles.ts

```diff
@@ -15,6 +16,7 @@ export class Particles {
     }
 
     public array: Particle[];
+    public spatialGrid : SpatialGrid;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected nospace before colon in property-declaration](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5342887)